CIP-1856/README.md

```diff
@@ -0,0 +1,149 @@
+---
+CIP: 1856
+Title: Extend NFT and FT metadata standard to support string localization

```

Comment:
```suggestion
Title: Extend metadata standard for translations
```
We need to keep these titles short and clear.  If there is a "token" that is not either an NFT or FT, please correct me.  Just suggesting this as an average between the PR title and the CIP title.  Long history of EE/CS standard names is to avoid punctuation wherever possible.